pubsub/api/Pubsub.Samples.Tests/PublishBatchMessageTest.cs

```diff
@@ -0,0 +1,51 @@
+﻿// Copyright 2020 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System.Collections.Generic;
+using System.Linq;
+using Xunit;
+
+[Collection(nameof(PubsubFixture))]
+public class PublishBatchMessageTest
+{
+    private readonly PubsubFixture _pubsubFixture;
+    private readonly PublishBatchedMessagesAsyncSample _publishBatchedMessagesAsyncSample;
+    private readonly PullMessagesAsyncSample _pullMessagesAsyncSample;
+
+    public PublishBatchMessageTest(PubsubFixture pubsubFixture)
+    {
+        _pubsubFixture = pubsubFixture;
+        _pullMessagesAsyncSample = new PullMessagesAsyncSample();
+        _publishBatchedMessagesAsyncSample = new PublishBatchedMessagesAsyncSample();
+    }
+
+    [Fact]
+    public async void PublishBatchMessagesAsync()
+    {
+        string topicId = "testTopicForBatchMessageCreation" + _pubsubFixture.RandomName();
+        string subscriptionId = "testSubscriptionForBatchMessageCreation" + _pubsubFixture.RandomName();
+
+        _pubsubFixture.CreateTopic(topicId);
+        _pubsubFixture.CreateSubscription(topicId, subscriptionId);
+
+        List<string> messageTexts = new[] { "Hello World!", "Good day.", "Bye bye." }.ToList();
```

Comment:
Check here and everywhere, I think I marked all, but just in case.
```suggestion
        List<string> messageTexts = new List<string> { "Hello World!", "Good day.", "Bye bye." };
```